packages/@adobe/spectrum-css-temp/components/button/index.css

```
@@ -157,6 +157,17 @@ governing permissions and limitations under the License.
     min-inline-size: unset;
     padding: var(--spectrum-global-dimension-size-65);
   }
+
+  &.is-pending {
+    .spectrum-CircleLoader {
```

```suggestion
  &.spectrum-Button--pending {
    .spectrum-Button-circleLoader {
```

just convention for naming, doubledash for states that aren't native, and since we use CSS modules, while copying the name will work, it can be difficult to tell what file it came from, so we use the namespace for the component followed by the subcomponent name

---

Basically SuitCSS style names. Done